plugins/hashsum_download/web_client/templates/hashsum_download_fileInfo.jade

```diff
@@ -0,0 +1,6 @@
+if file.has('sha512')
```

Comment:
As additional hashsum types are added, they'll have to be known ahead of time by the client, rather than looping through every member of an embedded field, I guess.